### DISCUSSION TITLE
add login with proton + an option to use SN for Proton Unlimited or higher customers

### DISCUSSION THREAD

**Discussion by r-i-porter** [standardnotes/forum#3588] [Suggestions]
Kinda all in the title. 

Implement sign in with proton like simple login did after their merger. 

Include a paid tier access to SN for proton unlimited subscribers (ideally highest just sayin) 

At least give us tentative target dates for these and guidance on how much work it might be to merge accounts or migrate if we use both SN and Proton after each phase of integration is completed.

**Comment by ElectricDryad** [comment]
I don't know if this is a suggestion for the SN team or the Proton team or both, but I am anxiously awaiting any news on this feature. I want to start using SN but I don't want to start an account or a subscription if it's being integrated with the Proton suite and I'll have to merge or something in x months. ANY information on what is going to happen or when would be amazing.

**Reply by TonyBShepherd** [reply]
Kinda feeling the same way (Visionary user), but decided to go ahead and subscribe (Professional) anyway. Made a Proton support request, was told the SN announcement (https://standardnotes.com/blog/joining-forces-with-proton) said that "Standard Notes continues to operate as a separate product with separate subscriptions". I replied that the frequent use of the word "join" implied otherwise :)
I'm just starting to explore the corners but for $10 a month I now have a secure, private place to keep my stuff. Having encrypted backups sent daily means no more fear of outages / hacks / mushroom clouds. Those backups will be kept in Proton and echoed.to a desktop (POP) email account too.
If Proton grants free access to me, I'll ask that my remaining SN subscription be applied to my Proton account. But even if not, I feel like I'm getting my money's worth. YMMV etc.
